tests/api-testing/tests/test_cipher.py

```
@@ -0,0 +1,984 @@
+        resp_create_cipher_simpleOO = session.post(
+            f"{base_url}api/{org_id}/cipher_keys", json=payload_simpleOO
+        )
+        print(resp_create_cipher_simpleOO.content)
+
+        if resp_create_cipher_simpleOO.status_code == 200:
+            break  # Exit the loop if the key is created successfully
+        elif resp_create_cipher_simpleOO.status_code == 400:
+            continue  # Key already exists, try again
+        else:
+            # assert False, f"Unexpected error: {resp_create_cipher_simpleOO.status_code} {resp_create_cipher_simpleOO.content}"
+            raise AssertionError(f"Unexpected error: {resp_create_cipher_staticAkeyless.status_code} {resp_create_cipher_staticAkeyless.content}")
```

Ruff (0.8.2)

Undefined name `resp_create_cipher_staticAkeyless`

(F821)